runtime/src/lib.rs

```diff
@@ -1901,6 +1901,7 @@ impl hermes_governance_platform::Config for Runtime {
 parameter_types! {
     // small value for test environment in order to check postponing expirations
     pub ExpirationsSchedulerMaxWeight: Weight = Perbill::from_percent(15) * BlockWeights::get().max_block; // TODO: order-book clarify
+    pub AlignmentSchedulerMaxWeight: Weight = Perbill::from_percent(50) * BlockWeights::get().max_block;
```

Comment:
we have substrate's scheduler `SchedulerMaxWeight` set to 80% of the block + `ExpirationsSchedulerMaxWeight` to 15%. Won't exceeding 100% (by 45%) be a problem?

Reply:
Probably, @vovac12 what di you think?

Reply:
Decided to set `SchedulerMaxWeight` to 50% and `AlignmentSchedulerMaxWeight` to 35%.
In this case `SchedulerMaxWeight` + `ExpirationsSchedulerMaxWeight` + `AlignmentSchedulerMaxWeight` = 100%